### PR TITLE
Prac 6 code review request

### PR DESCRIPTION
**Pull request by LIUCHUNH4N**
@tobden9
Can you help me check my practical 6 files?
If there is anything to improve, feel free give me any comment.

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -5,14 +5,19 @@
...
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
-state_code = input("Enter short state: ")
+for code, name in CODE_TO_NAME.items():
+    print(f"{code:<3} is {name}")
+
+
+state_code = input("Enter short state: ").upper()
```
Comment (tobden9) [inline prac_05/state_names.py:16]: the function input doesnt need to be called twice

File: prac_05/state_names.py [hunk 2]
```diff
@@ -5,14 +5,19 @@
...
 while state_code != "":
-    if state_code in CODE_TO_NAME:
+    try:
         print(state_code, "is", CODE_TO_NAME[state_code])
-    else:
+    except KeyError:
```
Comment (tobden9) [inline prac_05/state_names.py:20]: except KeyError needs to be placed inside the loop

File: prac_05/hex_colours.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+"""
+CP1404/CP5632 Practical
+Colors in a dictionary
+"""
+
+COLOR_CODES = {
```
Comment (tobden9) [inline prac_05/hex_colours.py:6]: the dictionary entries were aligined neatly

File: prac_05/kivy_layout.kv [hunk 4]
```diff
@@ -15,4 +15,7 @@ BoxLayout:
             text: "Down"
             on_press: app.handle_press(-1)
     BoxLayout:
-        id: names_box
```
Comment (tobden9) [inline prac_05/kivy_layout.kv]: double id is there

**Comment by tobden9** [comment]
Your codes are perfect

File: prac_05/state_names.py (hunk 1 above)
Comment (LIUCHUNH4N) [reply prac_05/state_names.py:16]: bro,they are practicals 5, maybe check my files in practical 6 :)

File: prac_06/car.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    def __init__(self, fuel=0, name="Car"):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 0
+        self.name = name
+
```
Comment (tobden9) [inline prac_06/car.py:15]: Consider adding checks to ensure that fuel and distance values are non-negative

**Review by tobden9 — COMMENTED** [review]
you could use more comments, 
good job

**Pull request merged**